routes/userProfile.js

```diff
@@ -0,0 +1,80 @@
+const router = require('express').Router()
+const db = require('../db')
+const authorization = require('../middleware/authorization')
+const calculateAge = require('../utils/calculateAge')
+const Constants = require('../utils/constant')
+
+const { months, validUUID } = Constants
+
+// send back current logged in user after token validation
```

Comment:
i love the comments, more comments more better but also code should comment itself